src/wasm-lib/kcl/src/parser/snapshots/kcl_lib__parser__parser_impl__snapshot_math_tests__b.snap

```diff
@@ -11,16 +12,19 @@ expression: actual
     "value": 1,
     "raw": "1",
     "start": 0,
-    "end": 1
+    "end": 1,
+    "moduleId": 0
```

Comment:
This might not be worth it, but: given that this will frequently be 0, we should at least consider making the serialization of this field skip if it's 0, and the deserialization default to 0. That way, the snapshots wouldn't all need to be updated, they'd be more terse and IMO readable, but they'd still work fine when moduleId is not 0.

Reply:
I frequently forget about this. I can try it.